src/homework5_task1.py

```diff
@@ -0,0 +1,27 @@
+""" Task1. Define a dict comprehension which returns a dictionary
+where the keys are numbers between 1 and n (both included)
+and the values are square of keys. n – function argument. Default is 20.
+"""
+n = 20
```

Comment:
it should be default value for input. Here there is no way to enter these arguments

Reply:
Does it mean that if there is no user input (Enter is pressed instead of entering number), it should be 20 by default? 
Sorry for misunderstanding.